tfx/components/statistics_gen/executor.py

```diff
@@ -107,26 +111,30 @@ def Do(self, input_dict: Dict[Text, List[types.Artifact]],
         stats_options.schema = schema
 
     split_uris = []
+    exclude_splits = exec_properties[EXCLUDE_SPLITS_KEY]
```

Comment:
use `.get()` instead to avoid mutating exec_properties.

Reply:
we actually defined exclude_splits to be an empty list if it's not provided so i can simplify line 117 directly.